www/include/configuration/configObject/servicegroup/DB-Func.php

```diff
@@ -333,14 +532,26 @@ function updateServiceGroupServices($sg_id, $ret = array(), $increment = false)
         : CentreonUtils::mergeWithInitialValues($form, 'sg_hgServices');
     for ($i = 0; $i < count($retTmp); $i++) {
```

Comment:
could you move the prepared statement of lines 538 and 548 before this loop. It will require lesser resources on DB